### ISSUE TITLE
use query params in view collection

### ISSUE THREAD

**Issue by amirrezasalimi** [pocketbase/pocketbase#2183]
Hi 
first of all thanks for this great project .
is there a way to use query parameters in viewCollection sql queries ? 
like this:
```sql
select count(*) from likes where post_id = "@query.post_id"
```

**Comment by ganigeorgiev** [comment]
No, view queries cannot be parameterized (you can also check https://github.com/pocketbase/pocketbase/discussions/2121, https://github.com/pocketbase/pocketbase/discussions/1948).

**Comment by ganigeorgiev** [comment]
An alternative approach to what you are trying to do is to use `GROUP BY`:

```
SELECT
  (ROW_NUMBER() OVER()) as id,
  count(*) as likes,
  post_id,
FROM likes
GROUP BY post_id
```

This will create a collection with 3 fields - `id`, `likes` and `post_id` and then you can request only the likes for the specific post with the list/search api, something like:

```
const result = await pb.collection("example").getFirstListItem(`post_id = "${yourPostId}"`)
```

**Comment by amirrezasalimi** [comment]
thanks , is querying with pocketbase sdk over view collections efficient ? i mean is that all in the end sql query ?
